packages/product/src/migrations/Migration20230707075414.ts

```diff
@@ -0,0 +1,11 @@
+import { Migration } from "@mikro-orm/migrations"
```

Comment:
question: Can we merge this with the other migration file?

Reply:
I didn't choose to merge them to prevent the current users from not being able to use it. But since we have the beta argument, we can still do that. Would you like me to merge them?

Reply:
Yes, I think we should merge them as long as we are in beta. One could even argue, that we will merge all migrations when the package reaches general availability.

Reply:
done